web/moduly/api/aktivityProgram.php

```diff
@@ -70,36 +74,59 @@
         'stitkyId'    => $stitkyId,
         // TODO: cenaZaklad by měla být číslo ?
         'cenaZaklad'  => intval($aktivita->cenaZaklad()),
-        'casText'     => $aktivita->zacatek()
-            ? $aktivita->zacatek()->format('G') . ':00&ndash;' . $aktivita->konec()->format('G') . ':00'
+        'casText'     => $zacatekAktivity
+            ? $zacatekAktivity->format('G') . ':00&ndash;' . $konecAktivity->format('G') . ':00'
             : '',
         'cas'         => [
-            'od' => $aktivita->zacatek()->getTimestamp() * 1000,
-            'do' => $aktivita->konec()->getTimestamp() * 1000,
+            'od' => $zacatekAktivity->getTimestamp() * 1000,
+            'do' => $konecAktivity->getTimestamp() * 1000,
         ],
         'linie'       => $aktivita->typ()->nazev(),
+        'vBudoucnu' => $aktivita->vBudoucnu(),
+        'vdalsiVlne' => $aktivita->vDalsiVlne(),
+        'probehnuta' => $aktivita->probehnuta(),
+        'jeBrigadnicka' => $aktivita->jeBrigadnicka(),
     ];
 
-    $vBudoucnu = $aktivita->vBudoucnu();
-    if ($vBudoucnu)
-        $aktivitaRes['vBudoucnu'] = $vBudoucnu;
-
-    $vdalsiVlne = $aktivita->vDalsiVlne();
-    if ($vdalsiVlne)
-        $aktivitaRes['vdalsiVlne'] = $vdalsiVlne;
-
-    $probehnuta = $aktivita->probehnuta();
-    if ($probehnuta)
-        $aktivitaRes['probehnuta'] = $probehnuta;
-
-    $jeBrigadnicka = $aktivita->jeBrigadnicka();
-    if ($jeBrigadnicka)
-        $aktivitaRes['jeBrigadnicka'] = $jeBrigadnicka;
+    if ($u) {
+        $stavPrihlasen = $aktivita->stavPrihlaseni($u);
+        switch ($stavPrihlasen) {
+            case StavPrihlaseni::PRIHLASEN:
+                $aktivitaRes['prihlasen'] = true;
+                break;
+            case StavPrihlaseni::PRIHLASEN_A_DORAZIL:
+                $aktivitaRes['prihlasenADorazil'] = true;
+                break;
+            case StavPrihlaseni::DORAZIL_JAKO_NAHRADNIK:
+                $aktivitaRes['dorazilJakoNahradnik'] = true;
+                break;
+            case StavPrihlaseni::PRIHLASEN_ALE_NEDORAZIL:
+                $aktivitaRes['prihlasenAleNedorazil'] = true;
+                break;
+            case StavPrihlaseni::POZDE_ZRUSIL:
+                $aktivitaRes['pozdeZrusil'] = true;
+                break;
+            case StavPrihlaseni::SLEDUJICI:
+                $aktivitaRes['sledujici'] = true;
+                break;
+        }
+
+        $aktivitaRes['slevaNasobic'] = $aktivita->slevaNasobic($u);
+
+        $aktivitaRes['vedu'] = $u && $u->organizuje($aktivita);
+        // TODO: argumenty pro admin
+        $aktivitaRes['zamcenaMnou'] = $aktivita->zamcenoUzivatelem($u);
+    }
+    $aktivitaRes['prihlasovatelna'] = $aktivita->prihlasovatelna();
+    $aktivitaRes['zamcenaDo'] = $aktivita->tymZamcenyDo()?->getTimestamp() * 1000;
```

Comment:
@Mirinek Tady jsi nepřenesl `->getTimestamp()`, takže to zkoušelo násobit object DateTime číslem a PHP se z toho zhroutilo.

Fixnul jsem a nasazuju, díky za úpravy.